tutorials/Window.getInfo.md

```diff
@@ -14,4 +14,11 @@ async function getInfo() {
 }
 
 getInfo().then(info => console.log(info)).catch(err => console.log(err));
+// {
```

Comment:
maybe we display the response as V1 doc: http://cdn.openfin.co/jsdocs/stable/tutorial-window.getInfo.html